src/bun.js/api/BunObject.zig

```diff
@@ -2484,6 +2503,14 @@ pub const Crypto = struct {
                 return .zero;
             }
 
+            switch (this.*) {
+                .evp => {},
+                .zig => |*inner| {
+                    inner.update(buffer.slice());
+                    return thisValue;
+                },
+            }
+
             this.evp.update(buffer.slice());
```

Comment:
can you move this to .evp =>

Reply:
wanted to keep the diff small for the first iteration to make easier to review and test